ui/components/nodeRenderers/math.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import katex from 'katex';
+
+
+export default function TeXMath({ children, docNode }) {
+  return (
+    <div id={docNode.identifier}>
+      { children }
```

Comment:
Do you think we should be accounting for children of `math` nodes? I'm thinking they'll always be a leaf.

If we're keeping this, should it be swapped down a line so that the content of this node is rendered above its children?

Reply:
Good point; they shouldn't have child elements.